tfac/predict.py

```diff
@@ -50,13 +50,14 @@ def evaluate_scaling():
     )
 
     for scaling, _ in by_scaling.items():
-        tensor, matrix, labels = get_scaled_tensors(scaling)
+        tensor, matrix, patient_data = \
+            form_tensor(scaling, drop_validation=True)
```

Comment:
The validation samples should be included during CMTF but then dropped after factorization for training the model. I think it's safest to not have a `drop_validation` option to ensure this is the case.